<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit_calc</class>
 <widget class="QDialog" name="deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Сумма влада(₽):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Налоговая ставка(%):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Процентная ставка(%):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Срок размещения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Периодичность выплат:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_vklad">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="stavka">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="srok">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nalog">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rashet">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Сумма налога:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Сумма на вкладе в концу срока:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="kapitalization">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>224</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Депозитный калькулятор</string>
   </property>
  </widget>
  <widget class="QComboBox" name="period">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>227</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждую неделю</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Пополнения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string>Снятие:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_sniatie">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_popolnenie">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_popolnenie">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sum_sniatie">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>150</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLabel" name="res_proc">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="res_full">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="res_sum">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  	border: 1px solid gray;
}

background-color : black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
